### PR TITLE
support to show the large icon for notification area

### PR DESCRIPTION
**Pull request by nfukuzaki**

### CODE REVIEW

File: AndroidPushPlugin/src/com/kii/cloud/unity/AbstractGcmIntentService.java [hunk 1]
```diff
@@ -224,13 +260,19 @@ protected void showNotificationArea(Context context, JSONObject message, boolean
...
 				.setContentText(notificationText);
-			if (icon != 0) {
-				notificationBuilder.setSmallIcon(icon);
-				Bitmap largeIcon = BitmapFactory.decodeResource(getResources(), icon);
-				if (largeIcon != null && Build.VERSION.SDK_INT >= 11) {
-					notificationBuilder.setLargeIcon(largeIcon);
+			if (smallIcon != 0) {
+				notificationBuilder.setSmallIcon(smallIcon);
+			}
+			if (Build.VERSION.SDK_INT >= 11 && largeIcon != 0) {
```
Comment (fkmkii) [inline AndroidPushPlugin/src/com/kii/cloud/unity/AbstractGcmIntentService.java]: NotificationCompat.Builderの内部でバージョンチェックは行われるので、if (largeIcon != 0)のみで大丈夫です

File: AndroidPushPlugin/src/com/kii/cloud/unity/AbstractGcmIntentService.java [hunk 2]
```diff
@@ -224,13 +260,19 @@ protected void showNotificationArea(Context context, JSONObject message, boolean
...
+				Bitmap largeIconBitmap = BitmapFactory.decodeResource(getResources(), largeIcon);
+				if (largeIconBitmap != null) {
+					notificationBuilder.setLargeIcon(largeIconBitmap);
 				}
 			}
+			Integer notificationColor = this.getNotificationColor(context);
+			if (Build.VERSION.SDK_INT >= 21 && notificationColor != null) {
```
Comment (fkmkii) [inline AndroidPushPlugin/src/com/kii/cloud/unity/AbstractGcmIntentService.java]: NotificationCompat.Builderの内部でバージョンチェックは行われるので、if (largeIcon != 0)のみで大丈夫です

**Comment by nfukuzaki** [comment]
@fkmkii Thanks

**Comment by ochtea** [comment]
Cool, it works well :sunglasses: 

- Android 5.0
screenshot_2015-09-08-18-25-16

- Android 4.1.3
screenshot_2015-09-08-18-26-46

Just noted I took above screenshots with sample application which has this PR plugin and following configuration.
- notificationTicker : `$.when`
- notificationTitle : `$.sender`
- notificationText : `$.payload`

**Pull request merged**